compiler/aarch64/codegen/ARM64BinaryEncoding.cpp

```diff
@@ -180,6 +184,12 @@ uint8_t *TR::ARM64ImmSymInstruction::generateBinaryEncoding()
                destination = (intptr_t)cg()->fe()->methodTrampolineLookup(cg()->comp(), symRef, (void *)cursor);
                TR_ASSERT_FATAL(cg()->comp()->target().cpu.isTargetWithinUnconditionalBranchImmediateRange(destination, (intptr_t)cursor),
                                "Call target address is out of range");
+
+#if defined(OSX)
+               // Regain permission for writing to the code buffer
+               // after calling methodTrampolineLookup()
```

Comment:
Could you amend this comment to include why this is the case?  i.e., that `methodTrampolineLookup()` may call `createTrampoline()` which will acquire/release write protection leaving the write permission disabled in this path.

Reply:
Updated the comment.